src/components/Select/AutosuggestMulti/__tests__/AutosuggestMulti.spec.js

```diff
@@ -0,0 +1,161 @@
+
+                expect(mockOnSelectionChange).not.toHaveBeenCalled();
+
+                wrapper.find('input').simulate('change', { target: { value: 'a' } });
+
+                expect(toJson(wrapper)).toMatchSnapshot();
+
+                wrapper.find('li').first().children().simulate('click');
+
+                expect(mockOnSelectionChange).toHaveBeenCalled();
+            });
+            it('should be called also when clicking on a suggestion the second time in a row', () => {
+                suggestionsList = SUGGESTIONS.slice(1, 20);
+                wrapper.setProps({ suggestions: suggestionsList });
+                setFocusOnAutosuggest();
+
+                expect(mockOnSelectionChange).not.toHaveBeenCalled();
+
+                wrapper.find('li').first().children().simulate('click');
+
+                expect(mockOnSelectionChange).toHaveBeenCalledTimes(1);
+
+                setFocusOnAutosuggest();
+                wrapper.find('li').first().children().simulate('click');
+
+                expect(mockOnSelectionChange).toHaveBeenCalledTimes(2);
+            });
+            it('should be called on deleting a suggestion by clicking on the x button next to it', () => {
+                selectedSuggestions = SUGGESTIONS.slice(0, 5);
+                wrapper.setProps({ selectedSuggestions });
+                setFocusOnAutosuggest();
+
+                expect(mockOnSelectionChange).not.toHaveBeenCalled();
+                expect(wrapper.find('SuggestionTag')).toHaveLength(selectedSuggestions.length);
+
+                wrapper.find('SuggestionTag').at(2).find('button').simulate('click');
+
+                expect(mockOnSelectionChange).toHaveBeenCalledWith(selectedSuggestions[2]);
+            });
+            it('should be called on deleting a suggestion by hitting backspace in the empty input field', () => {
+                selectedSuggestions = SUGGESTIONS.slice(0, 5);
+                wrapper.setProps({ selectedSuggestions });
+                setFocusOnAutosuggest();
+
+                expect(mockOnSelectionChange).not.toHaveBeenCalled();
+
+                wrapper.find('input').simulate('keyDown', { key: 'Backspace' });
+
+                expect(mockOnSelectionChange).toHaveBeenCalled();
+            });
+            it('should not be called on deleting a suggestion by hitting backspace in an input field with value', () => {
+                selectedSuggestions = SUGGESTIONS.slice(0, 5);
+                wrapper.setProps({ selectedSuggestions });
+                setFocusOnAutosuggest();
+                wrapper.find('input').simulate('change', { target: { value: 'driver' } });
+
+                expect(mockOnSelectionChange).not.toHaveBeenCalled();
+
+                wrapper.find('input').simulate('keyDown', { key: 'Backspace' });
+
+                expect(mockOnSelectionChange).not.toHaveBeenCalled();
+            });
+        });
+        it('should call onInputValueChange when typing into input field', () => {
```

Comment:
this and the next test are also in SelectBase, maybe we don't need to repeat them here, what do you think?

Reply:
I think better to keep it.
https://github.com/textkernel/oneui/pull/241#discussion_r398752386